allensdk/test/brain_observatory/behavior/test_behavior_ophys_session.py

```diff
@@ -196,11 +196,9 @@ def get_cell_specimen_table(self):
                 }, index=pd.Index(data=[10, 11], name="cell_specimen_id")
             )
 
-        def get_segmentation_mask_image(self):
-            data = roi_1  # useless image data here
-            spacing = (1, 1)
-            unit = 'index'
-            return ImageApi.serialize(data, spacing, unit)
+        def get_surface_2p_pixel_size_um(self):
```

Comment:
i assume this hard coded value is a temporary solution and will be replaced by a value pulled directly from the data?

Reply:
This function is used only for testing and is supposed to emulate a method that real data would have.

Reply:
ah i see, thanks. i probably could have figured that out myself given that 'test' is in the name of the script :)